pypeit/spectrographs/spectrograph.py

```diff
@@ -1146,7 +1146,11 @@ def get_rawimage(self, raw_file, det):
         """
         # Check extension and then open
         self._check_extensions(raw_file)
-        hdu = io.fits_open(raw_file)
+        try: 
```

Comment:
I think this can all be replaced with:
`hdu = io.fits_open(raw_file, output_verify='ignore', ignore_blank=True)`
since there is a try/except block to catch exactly this in `io.fits_open`

Reply:
replaced!